### ISSUE TITLE
Location Colouring/Grouping Review

### ISSUE THREAD

**Issue by LukePrior** [LukePrior/nbn-upgrade-map#367]
Linking on from #361 

I think I would like to first define all the unique situations for an address i.e.

```
FTTP
FTTP Upgrade Available
FTTP Upgrade Soon
HFC
FTTC
FTTC Upgrade Available
FTTB
FTTN
FTTN Upgrade Available
Wireless
Wireless Upgrade Available
Satellite
```

Then be able to dynamically join these into higher level groups (maybe store all the info in a json file).

So for example we might group FTTB & FTTN into a single item same with FW/Sat.

This could also enable toggles for showing available upgrades or not etc.

We could also store the colour info in this file so if people were very passionate they could customise to best suit what they want etc...

This would be quite a major change and not sure if it'll ever get to this point but this issue can serve as a place for ideas around improving the colouring/grouping generally.

**Comment by lyricnz** [comment]
Related to #318

**Comment by lyricnz** [comment]
FYI the rinseout side has a toggle which changes the dots from current-status to fttp-upgrade-status
https://nbn.rinseout.org/

**Comment by LukePrior** [comment]
Yeah since that site got updated a lot of our functionality was replaced but I think the historical record is extremely valuable

**Comment by LukePrior** [comment]
But I agree it's a good option to have and can work in a similar way

**Comment by lyricnz** [comment]
To respond to the original comment, here are the complete tech->upgrade:count tallies:

```
WIRELESS (748710)
  NULL: 744906
  FTTN: 473
  FTTP: 2239
  FTTC: 1069
  UNKNOWN: 23
SATELLITE (402342)
  NULL: 397581
  WIRELESS: 4506
  FTTP: 224
  FW: 6
  FTTN: 16
  UNKNOWN: 5
  SATELLITE: 3
  FTTC: 1
FTTN (3997187)
  FTTP: 3248459
  NULL: 748679
  FTTC: 45
  WIRELESS: 4
FTTP (3016590)
  FTTP: 176894
  NULL: 2839695
  SAT: 1
NULL (174818)
  NULL: 172505
  FTTP: 609
  UNKNOWN: 1704
HFC (2735336)
  NULL: 2735182
  FTTP: 154
FTTC (1511467)
  FTTP: 1473101
  NULL: 38366
FTTB (611235)
  NULL: 610473
  FTTP: 762
```

**Comment by BeauGiles** [comment]
@LukePrior is there any chance you could look at splitting up FW and SAT?
